annotations/security-schemes/README.md

```diff
@@ -11,14 +11,37 @@
 
 (see full [description](oauth-2-custom-settings.raml))
 
-### What's included?
+### What's included
```

Comment:
I think the `?` was meant to be

Reply:
markdown linter treats it as an error.

Reply:
Weird. I guess that's an issue with the linter you're using...